tests/bsim/bluetooth/host/l2cap/frags/src/main.c

```diff
@@ -0,0 +1,477 @@
+/*
+ * The goal of this test is to verify the buf->frags feature works with L2CAP
+ * credit-based channels.
+ *
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bstests.h"
+#include "common.h"
+
+#define LOG_MODULE_NAME main
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_DBG);
+
+CREATE_FLAG(is_connected);
+CREATE_FLAG(flag_l2cap_connected);
+
+#define SMOL_SIZE 10
+#define CHONK_SIZE 50
```

Comment:
What is a `chonk` in this context?

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/68428270/b9729ef9-ca44-4d01-94ab-3a5aa3837bf0)